src/app/components/list/list.service.js

```diff
@@ -41,16 +37,24 @@
 
     // Cache a list for future offline use
     List.prototype.cache = function () {
+      var lfusers = $localForage.instance('users');
+      var lflists = $localForage.instance('lists');
       var request = { limit: 50, offset: 0, sort: 'name'};
+      lflists.setItem(this._id, this);
```

Comment:
It would cause an error yes but the only consequence would be a message in the console... I don't see how it could cause a blocking error...